guides/assets/custom-header.html

```diff
@@ -1,20 +1,19 @@
+<!doctype html>
 <html>
 <head>
   <script src="/bower_components/webcomponentsjs/webcomponents-lite.min.js">
   </script>
-  <link rel="import" href="/bower_components/polymer/polymer.html">
   <link rel="import"
         href="/bower_components/paper-header-panel/paper-header-panel.html">
   <link rel="import"
         href="/bower_components/iron-flex-layout/iron-flex-layout.html">
   <link rel="import"
-        href="/bower_components/font-roboto/font-roboto.html">
+        href="/bower_components/font-roboto/roboto.html">
   <style>
-:root {
-    --primary-font-family: Roboto, 'Noto', 'Helvetica Neue', 
-                           Helvetica, Arial, sans-serif;
+body {
+    font-family: Roboto, 'Noto', 'Helvetica Neue', 
+                 Helvetica, Arial, sans-serif;
 }
-
 .paper-header {
   padding: 25px;
 }
```

Comment:
Do you mind indent the CSS inside the `<style>` block?  I think that would look cleaner if someone tries to look at the source.